ca/certificates.go

```diff
@@ -173,7 +168,6 @@ func (rca *RootCA) RequestAndSaveNewCertificates(ctx context.Context, paths Cert
 		if err == nil {
 			break
 		}
-		log.Warningf("error fetching signed node certificate: %v", err)
```

Comment:
Correct. Also none of these methods has `ctx`, which means that they won't be able to take advantage of the pretty format logging, and the module hierarchy.